src/ui-kit/form-controls/sam-sds-autocomplete/autocomplete-search/autocomplete-search.component.html

```diff
@@ -1,12 +1,12 @@
 <div sam-click-outside (clickOutside)="clickOutSide($event)" sam-tab-outside (tabOutside)="clickOutSide($event)">
   <div class="input-container">
-    <div role="combobox" [attr.id]="configuration.id + '-container'" [attr.aria-expanded]="showResults"
+    <div role="combobox" [attr.aria-label]="configuration.id + '-label'" [attr.id]="configuration.id + '-container'" [attr.aria-expanded]="showResults"
       [attr.aria-owns]="showResults ? configuration.id + '-listbox' : undefined" aria-haspopup="listbox">
       <input [disabled]="disabled" (keypress)="onkeypress($event)" (input)="textChange($event)"
         class="usa-input padding-right-3" [ngClass]="getClass()" #input
         [attr.aria-label]="configuration.ariaLabelText ? configuration.ariaLabelText : configuration.labelText"
         [attr.id]="configuration.id" type="text" (focus)="inputFocusHandler()" (keydown)="onKeydown($event)"
-        [(ngModel)]="inputValue" aria-autocomplete="list" [attr.placeholder]="configuration.autocompletePlaceHolderText"
```

Comment:
Can you explain me the reason why you're removing this..?

Reply:
![image](https://user-images.githubusercontent.com/49658682/122446552-cc8cd300-cf70-11eb-9d95-ed9a70d070b8.png)

It was removed because this attribute is throwing 508 issues in the AMP tool, and was raised as a defect.